core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/imagecatalog/ImageResponse.java

```diff
@@ -40,6 +40,9 @@
     @JsonInclude(NON_EMPTY)
     private StackDetailsJson stackDetails;
 
+    @JsonProperty("default")
+    private boolean defaultImage;
```

Comment:
JsonProperty should be default images ?

Reply:
This is a response json where each if the instances are marked with default true or false and based on that the ui decides which images to select by default.

Reply:
ok